esmvaltool/diag_scripts/clouds/clouds_taylor_show.ncl

```diff
@@ -0,0 +1,38 @@
+; CLOUDS_LIFRAC_SCATTER_POSTPROC
+; ############################################################################
+; Author: Axel Lauer (DLR, Germany)
+; ############################################################################
+; Description
+;   Read ice fraction from clouds_lifrac_scatter.ncl and plot results for
+;   cmip5/cmip6 model pairs defined below.
+;
+; Modification history
+;   20211113-lauer_axel: written.
+;
+; ############################################################################
```

Comment:
Is this file still needed?
The inpath is here hardcoded...

Reply:
Ups... this file should already have been deleted a long time ago. No idea why this slipped through. I removed the file now.